index.html

```diff
@@ -132,11 +132,144 @@
     <!--social media sidebae ends-->
 
     <!-- about starts -->
-    <section id="about" class="about">
-        <div class="container">
-            <h1 class="text-center">About Us</h1>
-            <hr>
 
+    <section id="about" class="about" style="background-color: 		#A9A9A9;">
```

Comment:
change the background color it is not in sync with the websites color scheme and add the property in the style.css file and not inline.

Reply:
Instead of merging with the about section create a new section that says team and use that. Leave the about us section as it is.